spec/grape/api_spec.rb

```diff
@@ -3817,7 +3817,9 @@ def serializable_hash
 </hash>
 XML
       end
-      it 'array' do
+      # Temporary skipped
+      # XML generation for an array behaves differently when Rails (or something else) is loaded
+      xit 'array' do
```

Comment:
We should fix this. How differently?

Reply:
I will try to figure out what causes this change in behaviour. Probably some 3rd party dependency when RuboCop and friends got updated.

Reply:
I think i've figured it out. The problem is that grape is not directly requiring `active_support/core_ext/array/conversions` that adds the `Array#to_xml` method. It has worked out of luck because activesupport internally loads the file when `active_support/time` is loaded.

Now in the latest 6.1 version of activesupport changes this internal loading, more precisely https://github.com/rails/rails/commit/12959dee0f19a8e050dd1236b031c2c690729905#diff-f805b1a36e26831e5eee78fb55549de44fb842f78443b178d5d5438105152017

I will revert this change and fix the issue in a separate PR